docs/document/website/versioned_docs/version-1.4.0/guides/version-conflicts-issue-solution.md

```diff
@@ -52,9 +52,15 @@ document.body.appendChild(button);
 
 ### Using CDN
 
-Users using Kintone UI Component through the CDN will always have the latest version.<br>
-Therefore, after the release of v1.4.0, some of your apps and plug-ins will not work correctly. <br>
-
+Users using Kintone UI Component through the CDN using the below link will always have the latest version.<br>
+Therefore, after the release of v1.4.0, some of your apps and plug-ins might not work correctly. <br>
+```sh
+https://unpkg.com/kintone-ui-component/umd/kuc.min.js
+```
+If you want to use a particular `Kuc` version, please specify the version number after the project name.
+```sh
+https://unpkg.com/kintone-ui-component@1.0.0/umd/kuc.min.js
+```
```

Comment:
I think you have a misunderstanding on CDN case.
unpkg automatically hosts kuc.min.js file after we release on npm. So the behavior and coding way is the same as UMD case above.
In case v1.4.0 is the latest version...
- This will be v1.4.0 kuc.min.js
  https://unpkg.com/kintone-ui-component/umd/kuc.min.js
- This will be v1.0.0 kuc.min.js
  https://unpkg.com/kintone-ui-component@1.0.0/umd/kuc.min.js

Thus, I think just linking to UMD is ok for CDN.

Sorry I didn't notice in the previous review.